tests/std/tests/GH_002711_Zc_alignedNew-/env.lst

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\usual_matrix.lst
+RUNALL_CROSSLIST
+PM_CL="/Zc:alignedNew-"

```

Comment:
I'm not sure if this is sufficient test. Does `move_only_function` actually work in this mode?

Reply:
Yes please